test/integration/test/philips_color_test.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+
+const chakram = require('chakram'), expect = chakram.expect;
+
+var Prepper = require('../lib/testPrepper.js');
+
+function translateHSB(h,s,b) {
+  var transHue = parseInt(65536 * (h % 360 / 360));
+  var transSat = parseInt(254 * s);
+  var transBri = parseInt(254 * b);
+
+  return {'h':transHue,'s':transSat,'b':transBri};
+}
+
+Prepper.makeSuite('Control lights locally',function(){
+
+  var lights;
+  var getterPayload = [{'kind': 'LightOn'}];
+
+  before(Prepper.foxboxManager.foxboxLogin);
+
+  it('check 3 bulbs are registered',function(){
+    
+    // collect all getters for the lightbulbs
+    return chakram.put(Prepper.foxboxManager.getterURL,getterPayload)
+    .then(function(listResponse) {
```

Comment:
I just remembered, for this kind of method we don't, above is the standard chakram convention for handling http response.